pkg/pillar/cmd/msrv/msrv.go

```diff
@@ -0,0 +1,639 @@
+// Copyright (c) 2024 Zededa, Inc.
```

Comment:
Is this derived from zedrouter/metadata.go? If so it makes sense to add 2024 to the existing copyright year.

Reply:
Done